src/dso_api/dynamic_api/routers.py

```diff
@@ -77,19 +90,20 @@ def _build_db_viewsets(self):
         generated_models = []
 
         datasets = {}
-        for dataset in Dataset.objects.db_enabled():  # type: Dataset
+        for dataset in _environment_enabled_datasets(
+            Dataset.objects.db_enabled()
+        ):  # type: Dataset
             dataset_id = dataset.schema.id  # not dataset.name!
             datasets[dataset_id] = dataset
             new_models = {}
 
-            for model in dataset.create_models():
-                logger.debug("Created model %s.%s", dataset_id, model.__name__)
-
-                # Register model in Django apps under Datasets application name,
-                #  because django requires fully set up app for model discovery to work.
-                register_model(dataset, model)
+            if dataset.enable_api:
```

Comment:
Heeft deze geen impact op WFS?
Denk ik models moeten geregestreerd worden onafhankelijk van API.

Reply:
Hmm die WFS zou ook naar `db_enabled` moeten kijken, of een aparte `enable_wfs` vlag. Ik zie dat dit dat niet goed gedaan heb.

De models zijn misschien wel nodig, mogelijk is er een relatie vanuit een andere dataset.